### DISCUSSION TITLE
Auto-Southwest Check-In v8.1

### DISCUSSION THREAD

**Discussion by jdholtz** [jdholtz/auto-southwest-check-in#315] [Announcements]
## New Contributors
* @sephamorr made their first contribution in https://github.com/jdholtz/auto-southwest-check-in/pull/314

**Concise Changelog**: [CHANGELOG.md](https://github.com/jdholtz/auto-southwest-check-in/blob/master/CHANGELOG.md#81-2024-11-03)
**Full Changelog**: https://github.com/jdholtz/auto-southwest-check-in/compare/v8.0...v8.1

<hr /><em>This discussion was created from the release <a href='https://github.com/jdholtz/auto-southwest-check-in/releases/tag/v8.1'>Auto-Southwest Check-In v8.1</a>.</em>

**Comment by foolio** [comment]
thank you for updating! the script has been throwing errors on 8.0 for the past month for me, 8.1 seems to work well so far
